### PR TITLE
typo: that, at line 29, inyour at line 151, friend at line 89.

### PR DESCRIPTION
**Pull request by Jalingpp**

### CODE REVIEW

**Comment by qqliang** [comment]
fix https://github.com/X-lab2017/oss101/issues/71

fix skip space typo: 'daysor' at line 149 to 'days or'.
fix extra letter typo: 'that,' at line 29 to 'that'.
fix extra space typo: 'mess age' at line 101 to 'message'.
fix skip space typo: 'inyour' at line 151 to 'in your'.
fix wrong letter typo: 'dpends' at line 19 to 'depends'.
fix wrong letter typo: 'friend' at line 89 to 'friends'.
team: 52265903004 52265903003 51255903082 51255903065

**Review by TieWay59 — CHANGES_REQUESTED** [review]
image
不应该有这么多行的修改。

File: Assignments/Assignment-typo.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Despite this, hackers have a reputation for meeting simple questions with what l
...
+We realize that there are many people who just want to use the software we write, and who have no interest in learning technical details. For most people, a computer is merely a tool, a means to an end; they have more important things to do and lives to live. We acknowledge that and don't expect everyone to take an interest in the technical matters that fascinate us. Nevertheless, our style of answering questions is tuned for people who *do* take such an interest and are willing to be active participants in problem-solving. That's not going to change. Nor should it; if it did, we would become less effective at the things we do best.
```
Comment (TieWay59) [inline Assignments/Assignment-typo.md:29]: 我们不包含修改逗号的。

File: .github/workflows/add-reaction.yaml [hunk 2]
```diff
@@ -18,6 +18,7 @@ jobs:
           "GiveMeAnApple99",
           "Goinggoinggoing",
           "qqliang",
+          "Jalingpp",
```
Comment (TieWay59) [inline .github/workflows/add-reaction.yaml:21]: 这个修改请放在其他 PR 。

**Review by TieWay59 — CHANGES_REQUESTED** [review]

**Comment by TieWay59** [comment]
如果你们实在找不到的话，我可以支持你们只需要两处typo，把那个不合适的去掉就好了。

**Pull request closed without merging**